protocol/rpcconsumer/consumer_relay_state_machine.go

```diff
@@ -59,20 +82,25 @@ func NewRelayStateMachine(
 	}
 
 	return &ConsumerRelayStateMachine{
-		ctx:                ctx,
-		usedProviders:      usedProviders,
-		relaySender:        relaySender,
-		protocolMessage:    protocolMessage,
-		analytics:          analytics,
-		selection:          selection,
-		debugRelays:        debugRelays,
-		tickerMetricSetter: tickerMetricSetter,
-		batchUpdate:        make(chan error, MaximumNumberOfTickerRelayRetries),
+		ctx:                     ctx,
+		usedProviders:           usedProviders,
+		relaySender:             relaySender,
+		protocolMessage:         protocolMessage,
+		originalProtocolMessage: protocolMessage,
+		analytics:               analytics,
+		selection:               selection,
+		debugRelays:             debugRelays,
+		tickerMetricSetter:      tickerMetricSetter,
+		batchUpdate:             make(chan error, MaximumNumberOfTickerRelayRetries),
 	}
 }
 
-func (crsm *ConsumerRelayStateMachine) SetRelayProcessor(relayProcessor *RelayProcessor) {
-	crsm.parentRelayProcessor = relayProcessor
+func (crsm *ConsumerRelayStateMachine) SetRelayRetriesManager(relayRetriesManager *lavaprotocol.RelayRetriesManager) {
```

Comment:
let's force both by:
setting initialized boolean to true when they are set, have only one function that needs to set both
if !initialized throw error on public funcs

Reply:
fixed 8beacb9fb16fdf20bfebe521a4d6d63bbd85412d